howso/visuals/visuals.py

```diff
@@ -581,3 +581,48 @@ def plot_interpretable_prediction(
 
     if len(figures) == 1:
         return figures[0]
+
+
+def plot_fairness_disparity(
+    fairness_results: dict,
```

Comment:
Consider using a `TypedDict` here to communicate what this might look like

Reply:
I put an example in to communicate what it will look like